backend/src/main/java/gov/cdc/usds/simplereport/db/model/FacilityLabTestOrder.java

```diff
@@ -0,0 +1,60 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class FacilityLabTestOrder extends EternalAuditedEntity {
+  @Column(nullable = false)
+  @NonNull
+  private UUID facilityId;
+
+  @Column(nullable = false)
+  @NonNull
+  private UUID labId;
+
+  @Column(nullable = false)
+  @NonNull
+  private String name;
```

Comment:
is the plan that this field will always be populated, even if the user did not choose to override the original name from the lab table?

Reply:
I am also confused by the Confirm button. Also about the behavior of the Reset buttons for the name and description fields. Do those reset to what is saved for the test order or what is on the original lab pulled from the LOINC API? Might be good to get additional UX input on the edit modal